### DISCUSSION TITLE
Fulltext search on widgetlist

### DISCUSSION THREAD

**Discussion by mzinatsa92** [OrchardCMS/OrchardCore#11593] [General]
Hi 
  Ive been trying to a do a fulltext search on contentitems with dynamic widgets. Has anyone done this successfully?

**Comment by Skrypt** [comment]
Sure, what is the issue?

**Comment by momentum-tunis** [comment]
@mzinatsa92 I added an update to the documentation that should show up in the next release of documenation

https://github.com/momentum-tunis/OrchardCore/blob/eaad1af1829362c7b9e16cbc08caa775dd7cea8c/src/docs/guides/implement-fulltext-search/README.md

Check the paragraph:

We can use the "Use custom full-text" to include the content of widgets or bags inside the full text search index. For example, for widgets inside a FlowPart we should use this Liquid script:

```
{% for contentItem in Model.Content.FlowPart.Widgets %}
  {{ contentItem | full_text_aspect }}
{% endfor %}
```

Or simply use:

`{{ Model.Content.FlowPart.Widgets | full_text_aspect }}`

**Comment by mzinatsa92** [comment]
Thank you so much, that worked :) 
I had to use 
{{ Model.ContentItem.Content.WidgetsListPart.Widgets | full_text_aspect }}
